metrics.go

```diff
@@ -17,13 +17,18 @@ import (
 type MetricType string
 
 const (
-	builtinMetricsPrefix = "juju"
+	BuiltinMetricPrefix = "juju"
 
 	// Supported metric types.
 	MetricTypeGauge    MetricType = "gauge"
 	MetricTypeAbsolute MetricType = "absolute"
 )
 
+// IsBuiltinMetric returns true if the metric key provided is in the builtin metric namespace.
```

Comment:
Standard phrasing would be something like:

    // IsBuiltinMetric reports whether the given metric key is in the
    // built-in metric name space.

("built-in" vs "builtin" and "name space" vs "namespace" are still
arguable, but "reports whether" is definitely better IMHO).